src/main/kotlin/no/nav/familie/ef/sak/vedtak/dto/VedtakDto.kt

```diff
@@ -98,12 +119,13 @@ fun VedtakDto.tilVedtak(behandlingId: UUID): Vedtak = when (this) {
 
 fun Vedtak.tilVedtakDto(): VedtakDto =
```

Comment:
Bør vi sende inn stønadstype her eller blir det overkill?

Reply:
Usikker
Vi vet den indirekt gjennom _type, så la til validering for det i `BeregnYtelseSteg` som sjekker att hvis man har 
InnvilgelseOvergangsstønad så sjekker vi att behandlingen er av type OS

Reply:
Men fra `Vedtak` til `VedtakDto` så vet du jo ikke det? Da sjekker vi jo på "null" på et felt

Reply:
Ah sånn ja. Ja vi snakket litt om det, landet ikke noe